docker-compose.yaml

```diff
@@ -57,14 +55,18 @@ services:
     depends_on:
       jaeger:
         condition: service_healthy
-
-
   jaeger:
     image: jaegertracing/all-in-one:latest
     ports:
-      - "6831:6831/udp"
-      - "16686:16686"
-      - "16685:16685"
+      - 5775:5775/udp
+      - 6831:6831/udp
+      - 6832:6832/udp
+      - 5778:5778
+      - 4318:4318
+      - 16685:16685
+      - 16686:16686
+      - 14268:14268
+      - 9411:9411
```

Comment:
is it necessary to expose ALL jaeger ports?

Reply:
Updated 😄